rfcs/2021-09-01-8547-accept-metrics-in-datadog-agent-source.md

```diff
@@ -144,8 +144,8 @@ The implementation would then consist in:
      without loss of information: `(Agent Sketch) -> (Vector) -> (Datadog intake)`. This RFC focus on ingesting sketch
      and not the rest of the flow.
 
-**Regarding the tagging issue:** A -possibly temporary- work-around would be to store incoming tags with the complete
-"key:value" string as the key and an empty value to store those in the extisting map Vector uses to store
+**Regarding the tagging issue:** A -possibly temporary- workaround would be to store incoming tags with the complete
```

Comment:
Most instances of `work around` in this repository were as a verb, but this is as a noun -- https://ell.stackexchange.com/questions/79334/why-is-workaround-written-as-one-word#:~:text=In%20the%201960s%2C%20people%20often,is%20almost%20always%20%22workaround%22.